app/components/canvas/canvas.js

```diff
@@ -140,11 +140,11 @@ class CanvasController {
     [this.canvas_, this.offscreenCanvas_].forEach(canvas => {
       canvas
           .attr({
-            width: this.artwork.width * this.backingStoreScale_, 
+            width: this.artwork.width * this.backingStoreScale_,
```

Comment:
stray change in this CL :)

I'd suggest looking here and spotting anything that doesn't belong:
https://github.com/romannurik/AndroidIconAnimator/pull/74/files